src/containers/PatientDetail/index.tsx

```diff
@@ -0,0 +1,310 @@
+    const [expanded, setExpanded] = useState<string | false>(false);
+    const [isActive, setIsActive] = useState<boolean>(false);
+    const [loading, setLoading] = useState<boolean>(false);
+    const [account, setAccount] = useState<Account>();
+    const [patient, setPatient] = useState<Patient>();
+
+    const params = useParams<{ id: string }>();
+    const accountId = params.id;
+
+    const convertDate = (input: string) => {
+        let date = input?.slice(0, 10);
+        const [year, month, day] = date?.split("-");
+        date = day + "/" + month + "/" + year;
+        return date;
+    };
+
+    const handleChange = (panel: string) => (event: React.SyntheticEvent, isExpanded: boolean) => {
+        setExpanded(isExpanded ? panel : false);
+    };
+
+    const handleCloseConfirmModal = async (
+        e: React.MouseEvent<HTMLButtonElement | MouseEvent>,
+        action: "CONFIRM" | "CANCEL"
+    ) => {
+        if (action === "CONFIRM") {
+            try {
+                const response = await axios.put("/accounts/change-status/" + accountId);
+                if (response.status === 200) {
+                    setIsActive(!isActive);
+                    showSnackbar({
+                        children: "Cập nhật trạng thái tài khoản thành công",
+                        variant: "filled",
+                        severity: "success",
+                    });
+                }
+            } catch (error) {
+                showSnackbar({
+                    children: "Cập nhật trạng thái tài khoản thất bại",
+                    variant: "filled",
+                    severity: "error",
+                });
+            }
+        }
+        setIsOpenConfirmModal(false);
+    };
+
+    const toggleBtnActive = () => {
+        setIsOpenConfirmModal(true);
+    };
+
+    const getAccountById = useCallback(async (accountId) => {
+        setLoading(true);
+        try {
+            const response = await axios.get("/accounts/" + accountId);
+            if (response.status === 200) {
+                const data: Account = response.data;
+                const convertedDob = convertDate(data.dob);
+                const accountRes: Account = { ...data, dob: convertedDob };
+                setIsActive(accountRes.active);
+                setAccount(accountRes);
+
+                const res = await axios.get("/patients/email/" + accountRes.email);
+                if (res.status === 200) {
+                    setPatient(res.data);
+                }
+            }
+        } catch (_error) {}
+    }, []);
+
+    useEffect(() => {
+        getAccountById(accountId);
+    }, [accountId, getAccountById]);
+
+    let genderIcon = (
+        <Icon color="primary" sx={{ fontSize: 30 }}>
+            male_outlined_icon
+        </Icon>
+    );
+    if (!loading && !account?.isMale) {
+        genderIcon = <Icon sx={{ color: pink[500], fontSize: 30 }}>female_outlined_icon</Icon>;
+    }
+    const profile = (
```

Comment:
profile, healthInfo and consultingHistory, i think u can seperate them to components and put thems to some file.